src/parameters.jl

```diff
@@ -15,5 +15,10 @@ const EXTENDED_NBHD_LVL_1 = 10
 
 const WRKS_RANDOM_FACTOR = 0.2
 
+# Minimal score to decrease nbhd days
+const OPT_LVL_1_PEN  = 100
```

Comment:
spacja za duzo xD

Reply:
Celowo xD
W sensie, jak mam w kodzie jakieś kilka przypisań stałych/parametrów koło siebie, to zrównuję im operatory przypisania, bo jest wtedy przejrzyściej IMHO.